client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-content/motion-content.component.ts

```diff
@@ -347,6 +350,22 @@ export class MotionContentComponent extends BaseMotionDetailChildComponent {
             }
             this._initialState = deepCopy(contentPatch);
             this.contentForm.patchValue(contentPatch);
+        } else {
+            const parentId = Number(this.route.snapshot.queryParams[`parent`]);
+            if (!this.titleFieldUpdateSubscription && parentId && !Number.isNaN(parentId)) {
+                this.titleFieldUpdateSubscription = this.repo
+                    .getViewModelObservable(parentId)
+                    .pipe(
+                        map(parent => parent?.number),
+                        distinctUntilChanged()
+                    )
+                    .subscribe(number => {
+                        this.contentForm.patchValue({
+                            title: this.translate.instant(_(`Amendment to`)) + ` ${number}`
```

Comment:
```suggestion
                            title: this.translate.instant(`Amendment to`) + ` ${number}`
```
calls to `translate.instant` are also extracted, you don't need the marker function